README.md

```diff
@@ -0,0 +1,93 @@
+# librespot
+
+This role installs [librespot](https://github.com/librespot-org/librespot) and creates a
```

Comment:
one full sentence per line

Reply:
That would exceed the column limit in this case.